### DISCUSSION TITLE
How to dump a json object containing invalid utf-8?

### DISCUSSION THREAD

**Discussion by zliucd** [nlohmann/json#3573] [Q&A] [answered]
Hi,

Is that possible to dump a json object containing invalid utf-8?  Currently I use ```try-catch``` to catch such cases, but I still would like to dump these objects.

Environment: OSX, Centos8.5
g++: 8.5.0
C++ std: 11 and 14

Thanks.

**Comment by nlohmann** [comment, marked as answer]
See https://json.nlohmann.me/api/basic_json/dump/ - which version are you using?

**Reply by zliucd** [reply]
Huge thanks!  I'm using 3.10.4
